pom.xml

```diff
@@ -0,0 +1,109 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <groupId>io.kooperate</groupId>
+  <artifactId>conifg-model-project</artifactId>
+  <packaging>pom</packaging>
+  <version>0.1-SNAPSHOT</version>
+  <name>Kooperate :: POM</name>
+
+  <properties>
+    <format.skip>false</format.skip>
+    <java.source>1.8</java.source>
+    <java.target>1.8</java.target>
+
+    <!-- Plugins -->
+    <version.maven-compiler-plugin>3.8.0</version.maven-compiler-plugin>
+    <version.maven-failsafe-plugin>3.0.0-M3</version.maven-failsafe-plugin>
+    <version.maven-surefire-plugin>3.0.0-M3</version.maven-surefire-plugin>
+
+    <!-- Formatting Plugins -->
+    <version.formatter-maven-plugin>2.12.2</version.formatter-maven-plugin>
+    <version.impsort-maven-plugin>1.6.2</version.impsort-maven-plugin>
+
+    <!-- Formatting Plugins -->
+    <version.formatter-maven-plugin>2.12.2</version.formatter-maven-plugin>
+    <version.impsort-maven-plugin>1.6.2</version.impsort-maven-plugin>
+
+    <!-- Release Plugins -->
+    <version.maven-gpg-plugin>1.6</version.maven-gpg-plugin>
+    <version.maven-enforcer-plugin>1.3.1</version.maven-enforcer-plugin>
+    <version.maven-javadoc-plugin>2.10.3</version.maven-javadoc-plugin>
+    <version.maven-source-plugin>2.4</version.maven-source-plugin>
```

Comment:
I don't have a strong opinion about this, but I prefer the format `<dependency.name.version>` instead of `<version.dependency.name>`. 
Let's see what others think about this.

Reply:
I use to be really opinionated and pro `xxxx.version` based on the argument that the maven itself is using this convention in `project.version`. Nowdays I don't really mind but I find the version prefix convinient.

Reply:
@manusa @aureamunoz wdyt?

Reply:
:)) TL;DR I really don't care.

I prefer the version.xxx approach because I like to identify the purpose of the properties (I also tend to keep them sorted alphabetically ---> but only when there is some sort of hierarchy on the naming ---> usually Maven plugin properties do have a prefix (but not always)). When the number of properties grow, this helps me keep things clear.

Anyway, I seem to be alone on this boat, and usually most projects go with the xxx.version approach. So, I don't really mind. For me the important thing is to keep things consistent regardless of the chosen approach (OCD).

Reply:
I don't care either. The `version.xxxx` is ok for me.